extensions/fairifier-rdf-extension/dtls/fairifier/DetectFileFormatCommand.java

```diff
@@ -155,22 +133,4 @@ private String parseFullFile(FileItem file, String baseuri) {
         return format;
 
     }
-
-    private String getFormatFromUrl(String url)
```

Comment:
This was unused so i removed it, the backend handles the parsing from url in a different call